lib/use-cases/send-signpost-email.js

```diff
@@ -0,0 +1,16 @@
+export default class SendSignpostEmail {
```

Comment:
This is where I probably don't understand Clean Architecture well enough...
![image](https://user-images.githubusercontent.com/43747286/112459195-7c2d1d00-8d55-11eb-913d-751e131cfd42.png)


Is use-case having a use-case within its dependencies valid? Shouldn't it be one of the two?:
1. Controller (endpoint) calling 2 use-cases (create-converstation & send-signpost)
2. `CreateConverstation` use-case calling 2 gateways (create-converstation & notify)